modules/yoastseo/lib/Requirement/Readability_Analysis.php

```diff
@@ -110,7 +188,30 @@ public function get_option($option_name)
     public function get_current_status($post, $option_value)
     {
         $score = (int)get_post_meta($post->ID, '_yoast_wpseo_content_score', true);
+        $rank  = $this->rank_from_numeric_score($score);
+
+        return $rank == $option_value;
+    }
+
+    /**
+     * Returns a rank for a specific numeric score.
+     *
+     * @param int $score The score to determine a rank for.
+     *
+     * @return self
+     */
+    public static function rank_from_numeric_score($score)
+    {
+        // Set up the default rank.
+        $rank = self::BAD;
+
+        foreach (self::$ranges as $rank_index => $range) {
+            if ($range['start'] <= $score && $score <= $range['end']) {
+                $rank = $rank_index;
+                break;
+            }
+        }
 
-        return $score >= '71';
```

Comment:
You already have a start and end values, use them to check instead of looking for an exact value here.